apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -93,26 +88,29 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
         <>
             {coinType && <PageTitle title={coinSymbol} back="/tokens" />}
 
-            <div className={st.container} data-testid="coin-page">
-                {showError && error ? (
-                    <Alert className={st.alert}>
+            <div
+                className="flex flex-col h-full flex-1 flex-grow items-center"
+                data-testid="coin-page"
+            >
+                {error instanceof Error ? (
```

Comment:
Why not just `!!error`? or check `isError`?

Reply:
Error is does not have a message without that. Unless the hook returns the type and Error, error.message is undefined.

Reply:
Yeah, it doesn't correctly, because the error could be any type (although this is changing in v5 of tanstack query).